cmake/Platform/Common/RuntimeDependencies_common.cmake

```diff
@@ -64,7 +64,15 @@ function(ly_get_runtime_dependencies ly_RUNTIME_DEPENDENCIES ly_TARGET)
         unset(dependencies)
         get_target_property(dependencies ${ly_TARGET} LINK_LIBRARIES)
         if(dependencies)
-            list(APPEND link_dependencies ${dependencies})
+            foreach(dependency IN LISTS dependencies)
```

Comment:
this needs to be done after the if over the full list of `link_dependencies`
Otherwise, if a target has a 3rdParty with this flag in an `INTERFACE` build dependency, it would still be copied.

Reply:
From the [CMake docs on Interface Libraries](https://cmake.org/cmake/help/latest/command/add_library.html#interface-libraries):
> An `INTERFACE` library target does not compile sources and does not produce a library artifact on disk.

How do we copy something that isn't there?

Reply:
Oh, I see, those can come from the transitive libraries from `INTERFACE_LINK_LIBRARIES`

Reply:
Fixed.